### PR TITLE
COR-356 Add validation for folders

### PR DESCRIPTION
**Pull request by ludydoo**

### CODE REVIEW

File: pkg/server/public/handlers/folder/create.go [hunk 1]
```diff
@@ -25,6 +27,13 @@ func (h *Handler) Create() http.HandlerFunc {
 
 		folder.ID = uuid.NewV4().String()
 
+		if errList := validation.ValidateFolder(&folder); errList.HasAny() {
```
Comment (ludydoo) [inline pkg/server/public/handlers/folder/create.go:30]: Change HasAny to !IsEmpty

**Review by internetti — COMMENTED** [review]
Is there a way to adapt the error messages? when the folder name is too short, the message is "Too long: must have at minimum 3 bytes", I think "characters" would be better than "bytes"

**Comment by ludydoo** [comment]
@internetti we would have to modify the `validation.TooLong` / `validation.TooShort` methods. Perhaps we could tackle this in another tix ?

**Pull request merged**